tool-plugins/vscode/.vscodeignore

```diff
@@ -6,6 +6,7 @@ src/
 target/
 docs/
 extractedDistribution/
+resources/composer
```

Comment:
Do we need this? If the assets are coming from the distribution will we need to create a resources/compoer directory?

Reply:
@roonyh good catch, yeah we don't need it. I guess I needed it because composer files were already there from existing builds. But if we do a clean build this shouldn't be needed.

Reply:
Got it 👍